stock_by_warehouse_sale/i18n/es.po

```diff
@@ -0,0 +1,42 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* stock_by_warehouse_sale
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0+e\n"
```

Comment:
@hugho-ad Maybe an update of version in here 🙈 That was missing from 12 too hahaha